dart-2.16/pubspec.yaml

```diff
@@ -6,8 +6,8 @@ publish_to: 'none'
 environment:
   sdk: '>=2.16.0 <3.0.0'
 
-dependencies: 
-  dart_appwrite: ^7.1.0
-      
+dependencies:
+  dart_appwrite: ^8.0.0
+
```

Comment:
It looks like the dart_appwrite package [version 7 and above require dart 2.17](https://pub.dev/packages/dart_appwrite/versions), so let's downgrade the versions for everything under dart 2.17 to ^6.0.1

Reply:
Hi, I've pushed a commit with the required changes